Test/Utils/Initialize-Testbed.ps1

```diff
@@ -0,0 +1,34 @@
+function Get-DockerfilesPath {
+    return "C:\DockerFiles"
+}
+
+function Get-DNSDockerName {
+    return "python-dns"
+}
+function Initialize-Testbeds {
+    Param (
+        [Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions
+    )
+    foreach($Session in $Sessions) {
+        Write-Log "Downloading Docker images."
+        Invoke-Command -Session $Session -ScriptBlock {
+            docker pull microsoft/windowsservercore
+            docker pull microsoft/nanoserver
+        } | Out-Null
```

Comment:
We should check error code and write-log something, we should probably write pull output to stdout

Reply:
Good point, I'll change it here and below